### ISSUE TITLE
Questions about splitting the dataset

### ISSUE THREAD

**Issue by iskiku** [open-mmlab/mmdetection#11931]
I have a question, when training an object detection model, is it necessary to split the data set into train set, validation set and test set? (Sometimes it seems that others only split the dataset into train set and validation set)
